prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -83,6 +83,28 @@ gke_job_template: &gke_job_template
             memory: 200Mi
             cpu: 80m
 
+gke_upgrade_job_template_for_debug: &gke_upgrade_job_template_for_debug
+  decorate: true
+  path_alias: github.com/kyma-project/kyma
+  max_concurrency: 10
+  spec:
+    containers:
+      - <<: *base_image_k15
```

Comment:
Let's use `base_image_k16` cause the GKE cluster where Kyma will be installed would use k8s 1.16

Reply:
jupp, your right... copied from an outdated version